src/v2/codegen/object.rs

```diff
@@ -233,6 +236,16 @@ pub struct ObjectField {
     pub child_req_fields: Vec<String>,
 }
 
+pub fn to_snek_case(name: &str) -> String {
```

Comment:
did you mean snake_case?

Reply:
for some reason the old code also used to_snek_case from https://docs.rs/heck/0.3.0/heck/trait.SnekCase.html , so I named it also like this. But after looking into this crate, we could switch to use to_snake_case everywhere, as snek_case is only an alias.

Reply:
I changed all `snek_case` to `snake_case`